CorsixTH/Lua/dialogs/confirm_dialog.lua

```diff
@@ -31,7 +32,13 @@ local middle_frame_height = 11
 local bottom_frame = 359
 local text_width = 153
 
-function UIConfirmDialog:UIConfirmDialog(ui, text, callback_ok, callback_cancel)
+--! Initialise the Confirmation Dialog
+--!param ui The UI
+--!param is_error (boolean) set whether this dialog is a result of an error (true) or not (false)
```

Comment:
Why are we abstracting the effect from the name here? I think I prefer `must_pause` as that's the real effect not just an implementation detail.